modules/node-certificates.adoc

```diff
@@ -0,0 +1,19 @@
+
+// Module included in the following assemblies:
+//
+// * authentication/certificates/certificate-reference.adoc
+
+[id="node-certificates_{context}"]
+= Node certificates
+
+[discrete]
+== Purpose
+
+Node certificates are signed by the cluster; they come from a certificate
```

Comment:
I believe this part should be the most robust, due to the fact customers are very concern about it (most likely due to OCP 3.x expirations)
So it shoud be described that first is the boostrap and then we have new certs after ~ 24 hours that are valid for 20 days? (ocp 4.2?) and they are autorotated... when? if left >= 20% time of lifetime?
We should also mention who signed them (I see Issuer: CN = kube-csr-signer_@1586557208) and where it's located (what namespace and what secrets/configmaps?)